tensorflow/python/framework/load_library.py

```diff
@@ -175,7 +177,7 @@ def register_filesystem_plugin(plugin_location):
     RuntimeError: when unable to load the library.
   """
   if os.path.exists(plugin_location):
-    py_tf.TF_RegisterFilesystemPlugin(plugin_location)
+    py_tf.TF_RegisterFilesystemPlugin(plugin_location, plugin_function)
```

Comment:
We should an if to convert `plugin_function=None` to the default `TF_InitPlugin`.